### PR TITLE
feat(update): remove create or update

### PR DESCRIPTION
**Pull request by toumorokoshi**
create or update introduces a new pattern in addition to the existing flow of:

1. getting a resource.
2. if NOT_FOUND, send a create.
3. if found, send an update.

In the future, this could be re-added as an Apply method, which would map to the appropriate REST method of PUT.

## 🍱 Types of changes

What types of changes does your code introduce to AEP? _Put an `x` in the boxes
that apply_

- [x] Enhancement
- [ ] [New proposal](https://aep.dev/1#workflow)
- [ ] Migrated from google.aip.dev
- [ ] Chore / Quick Fix

## 📋 Your checklist for this pull request

Please review the [AEP Style and Guidance](https://aep.dev/style-guide) for
contributing to this repository.

### General

- [x] Basic [Guidance](https://aep.dev/style-guide#guidance) is met.
- [x] Ensure that your PR
      [references AEPs](https://aep.dev/style-guide#referencing-aeps)
      correctly.
- [x] [My code has been formatted](https://aep.dev/contributing#formatting)
      (usually `prettier -w .`)

💝 Thank you!

### CODE REVIEW

**Comment by rambleraptor** [comment]
I'm in favor of this as long as the goal is to add an Apply method. If we don't add Apply long-term, this feels useful.

**Review by rambleraptor — APPROVED** [review]

**Comment by toumorokoshi** [comment]
Feedback offline from @rofrankel is to wait until apply is merged (issue tracked in #110)

**Comment by toumorokoshi** [comment]
blocked on #248 - once that is merged, this will also merge.

**Pull request merged**
